packages/dropdown/src/__stories__/Dropdown.new-stories.js

```diff
@@ -21,7 +21,10 @@ export default {
   },
 };
 
-const Template = (args) => <Dropdown {...args} />;
+const Template = (args, { globals: { colorScheme, density } }) => {
+  console.log(density);
```

Comment:
left over console, is this for testing?

Reply:
left this in for now for when we need to play off themes for icon resizing.